pom.xml

```diff
@@ -64,16 +64,16 @@
     <properties>
         <revision>2.22</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.277.1</jenkins.version>
+        <jenkins.version>2.303.1</jenkins.version>
```

Comment:
2.303 does seem like a big bump, but it does seem like the majority of users are on that.

Reply:
FTR: https://github.com/jenkinsci/jenkins/pull/5320 is first in 2.303.x as an LTS line, so this is not arbitrary.